code/modules/hydroponics/spreading/spreading_response.dm

```diff
@@ -61,7 +61,7 @@
 			unbuckle()
 		else
 			user.setClickCooldown(100)
-			var/breakouttime = rand(600, 1200) //1 to 2 minutes.
+			var/breakouttime = rand(30 SECONDS, 1 MINUTE)
```

Comment:
blessed change

Reply:
could even make it more reasonable tbh, 20 seconds to a minute. You end up standing there for so long.